activestorage/app/models/active_storage/blob.rb

```diff
@@ -44,21 +44,25 @@ def find_signed(id)
     end
 
     # Returns a new, unsaved blob instance after the +io+ has been uploaded to the service.
-    def build_after_upload(io:, filename:, content_type: nil, metadata: nil)
+    # Set the +content_type+ and +extract_content_type_from_io+ to +false+ to manually
+    # specify the content type rather than using automatic content type inference.
+    def build_after_upload(io:, filename:, content_type: nil, metadata: nil, extract_content_type_from_io: true)
```

Comment:
I’d like to name this argument `identify` here and in `create_after_upload` for consistency with the [`identify`](http://api.rubyonrails.org/classes/ActiveStorage/Blob/Identifiable.html#method-i-identify) method.

Reply:
Renamed to identify in this commit 4f9defb42cf7c543906a7de72c7b6f91fa5058d4